app/models/template.rb

```diff
@@ -146,6 +145,32 @@ def import_oses(options)
     end
   end
 
+  def import_taxonomies(options)
+    process_organizations options
+    process_locations options
```

Comment:
I think this adds unnecessary layer, could we jusř use process_organizations a locations directly?